src/main/webapp/portal-core/js/portal/layer/querier/QueryTargetHandler.js

```diff
@@ -120,19 +120,54 @@ Ext.define('portal.layer.querier.QueryTargetHandler', {
                 continue;
             }
 
-
             point = Ext.create('portal.map.Point', {
                 latitude : queryTargets[i].get('lat'),
                 longitude : queryTargets[i].get('lng')
             });
 
+
             var shortTitle = cswRecord.get('name');
-            var maxTitleLength = 90;
-//            if (onlineResource && onlineResource.get('name')) {
-//                shortTitle += ' - ' + onlineResource.get('name');
-//            }
-            shortTitle += ' - ' + cswRecord.get('adminArea');
+            
+            // abbreviate the service provider name
+            var provider = cswRecord.get('contactOrg');   
+            
+            // list of service providers 
+            var SERVICE_PROVIDERS = [
+                'CSIRO',
+                'Geoscience Australia',
+                'Northern Territory',
+                'Queensland',
+                'South Australia',
+                'Tasmania',
+                'Victoria',
+                'Western Australia'
+            ];
+            
+            for (var i = 0; i < SERVICE_PROVIDERS.length; i++) {                
+                if (String.prototype.includes) {
```

Comment:
Ah yes, good old varied string compatibility. For compatibility sake I'd either stick to String.indexOf (which works on every browser) or add String.prototype.includes to src/main/webapp/portal-core/js/portal/Compatibility.js